library/src/interactor_impl.cxx

```diff
@@ -688,6 +688,12 @@ bool interactor_impl::recordInteraction(const std::string& file)
     return false;
   }
 
+  if (!vtksys::SystemTools::FileExists(file))
+  {
+    log::error("Interaction log file to record interactions to does not exist", file);
```

Comment:
Rather than checking if the file exists,I think we should check if the path is writable or not, because when new interaction tests are created, we specifiy filenames that may not exist yet.

Reply:
I am working on this